t/13-Config/30-min_app_githooks_version.t

```diff
@@ -0,0 +1,55 @@
+#!perl
+
+use strict;
+use warnings;
+
+use App::GitHooks;
+use App::GitHooks::Test;
+use App::GitHooks::Utils;
+use Test::Exception;
+use Test::FailWarnings -allow_deps => 1;
+use Test::Requires::Git;
+use Test::More;
+
+plan(tests => 6);
+
+# Require git.
+test_requires_git( '1.7.4.1' );
+
+App::GitHooks::Test::ok_reset_githooksrc(
+    content => ''
+);
+
+my $app = App::GitHooks->new(
+    arguments => [],
+    name      => 'commit-msg',
+);
+
+ok($app->get_config(),'config is retrieved when no min_app_githooks_version is specified');
```

Comment:
`new()` calls `get_config()` so it's technically redundant when checking for the exception. On the other hand, explicitly calling the method where the check is makes the test less brittle. I would just be consistent across all three tests (either call `get_config()` everywhere or remove it entirely).